packages/components/src/combobox-control/stories/index.js

```diff
@@ -266,17 +270,19 @@ const mapCountryOption = ( country ) => ( {
 
 const countryOptions = countries.map( mapCountryOption );
 
-function Template( args ) {
+function Template( { onChange, ...args } ) {
 	const [ value, setValue ] = useState( null );
 
 	return (
 		<>
 			<ComboboxControl
 				{ ...args }
 				value={ value }
-				onChange={ setValue }
+				onChange={ ( ...changeArgs ) => {
+					setValue( ...changeArgs );
+					onChange( ...changeArgs );
```

Comment:
Is this necessary? (It would probably be if the story was written in TypeScript?)

```suggestion
					onChange?.( ...changeArgs );
```

Reply:
You're right, good one!